confirmation/models.py

```diff
@@ -85,7 +86,7 @@ def create_confirmation_link(
     obj: Union[Realm, HasRealmObject, OptionalHasRealmObject],
     confirmation_type: int,
     *,
-    validity_in_days: Optional[int] = None,
+    validity_in_days: Union[Optional[int], NoValueSent] = NoValueSent(),
```

Comment:
This `NoValueSent` approach is great!

Reply:
Though maybe `UnspecifiedValue` would be a better name for this class, if we're also using it some non-API contexts?

Reply:
Yes `UnspecifiedValue` looks better. Updated.